recipe/meta.yaml

```diff
@@ -7,9 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ pkg_name }}-{{ version }}.zip
```

Comment:
@jreback I removed the `fn`. It is better to let `conda` find the file name from the URL 99.99% of time (rare cases need the `fn`).